packages/components/src/SidePanel/SidePanel.scss

```diff
@@ -6,6 +6,8 @@
 /// Side panel toggle button color
 /// @type Color
 $tc-side-panel-toggle-color: tint($brand-primary, 75);
+$list-item-hover-color: darken($brand-primary, 2%);
```

Comment:
It's not the background color of the guideline ( maybe in another pr ? )
Also, the selected background color is not the same in the guideline
Would we do another pr for both background color & this one is only about toggle button ?

Reply:
@talend-sromain i'll update all sidebar colors in separate PR. Because if i update only toggle button (to use white highlight) it will look bad.

Reply:
@vyshkov indeed, it would... can you add me as reviewer also on this other PR ? Thanks

Reply:
@mloussot, ok, i'll add you

Reply:
Thanks